src/ice_shelf/MOM_ice_shelf_dynamics.F90

```diff
@@ -2393,14 +2446,21 @@ subroutine calc_shelf_driving_stress(CS, ISS, G, US, taudx, taudy, OD)
           endif
         endif
 
-        sx_e(i,j) = (-.25 * G%areaT(i,j)) * ((rho * grav) * (ISS%h_shelf(i,j) * sx))
-        sy_e(i,j) = (-.25 * G%areaT(i,j)) * ((rho * grav) * (ISS%h_shelf(i,j) * sy))
+        if (CS%max_surface_slope>0) then
+          scale = min(CS%max_surface_slope/sqrt(sx**2+sy**2),1.0)
```

Comment:
Please add parentheses around the squares, to read `scale = min(CS%max_surface_slope/sqrt((sx**2)+(sy**2)),1.0)` so that this expression does not break rotational symmetry when Fused-multiply-adds (FMAs) are enabled.
  The rest of the MOM6 code outside of the ice-shelf satisfies rotational symmetry when FMAs are enabled.  I am noting other places where the code adjacent to the changes in this PR (e.g., on lines 3095 and 3125) where parentheses would need to be added to give rotationally symmetric answers with an active ice-shelf and enabled FMAs.  Such changes would not impact other groups who are not using the MOM6-ice-shelf, so it is not urgent that we add these parentheses elsewhere, but they should be added before we starting using the ice-shelf code with FMAs.